docs/src/stream.rst

```diff
@@ -26,7 +26,11 @@ Data types
 
 .. c:type:: uv_write_t
 
-    Write request type.
+    Write request type. Careful attention must be paid for resuing objects of 
+    this type. When a stream is in non-blocking mode, write requests sent 
+    with ``uv_write`` will be queued, resuing objects at this point is an
```

Comment:
Can you end the sentence after queued, and start a new sentence with Reusing (spelling again).